keps/sig-network/2681-pod-host-ip/README.md

```diff
@@ -364,6 +364,7 @@ N/A
 ## Implementation History
 
 - 2021-05-06: Initial KEP
+- 2023-08-15: Alpha release with kuberentes 1.28
```

Comment:
Commenting here, since I can't comment on untouched lines:

L264: Actually, it changes default behavior of k8s itself by automatically propagating HostIPs field. Would be worth to add.

L276: Have those been added? If not, can you please prioritize it?

L287: when the field is being set? I'm assuming that when the pod is actually touched?
I'm asking in the context of feature enablement - let's say I enable the feature (switch it on kubelet) - will it immediately update all the running pods to set the HostIPs field?

L291: Please update that it will be done before switching the feature gate for Beta. Please ensure that it will happen and update the KEP accordingly.
Ideally - it would be great to update it now with the plan of how exactly it will be tested - this is a nice example:
https://github.com/kubernetes/enhancements/pull/3658/files

L318: Please update - we don't want to leave TBD. That said, I'm probably fine with not having any specific SLI here.

L357: there was new question added to PRR:
https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template#can-enabling--using-this-feature-result-in-resource-exhaustion-of-some-node-resources-pids-sockets-inodes-etc
please update